contrib/ruby_event_store-rom/Gemfile

```diff
@@ -7,4 +7,7 @@ gem "mysql2", ">= 0.5.3"
 gem "pg", ">= 1.2.2"
 gem "ruby_event_store", path: "../../ruby_event_store"
 gem "sqlite3", "1.4.2"
-gem "activesupport"
```

Comment:
Gemfile is purely for test. We're not using bundler groups at all anywhere in the  RES codebase. 

You can have a look on ROM dependencies at:
https://github.com/RailsEventStore/rails_event_store/blob/be89cf85a50461b2a4df5835e5b80c2672267b9f/contrib/ruby_event_store-rom/ruby_event_store-rom.gemspec#L29-L36